test/test-fs.c

```diff
@@ -1323,6 +1323,61 @@ TEST_IMPL(fs_chmod) {
 }
 
 
+TEST_IMPL(fs_unlink_readonly) {
+  int r;
+  uv_fs_t req;
+  uv_file file;
+
+  /* Setup. */
+  unlink("test_file");
+
+  loop = uv_default_loop();
+
+  r = uv_fs_open(loop, &req, "test_file", O_RDWR | O_CREAT,
+    S_IWUSR | S_IRUSR, NULL);
```

Comment:
Done. S_ below uv, as with most other calls to uv_fs_open in that file. Let me know if that's not it.